client/src/components/RecordButton.js

```diff
@@ -47,12 +48,17 @@ const RecordButtonIcon = styled(MicrophoneIcon)`
 
 function RecordButton(props) {
   return (
-    <RecordButtonGradientWrapper {...props}>
-      <RecordButtonInput>
-        <RecordButtonIcon {...props} />
+    <RecordButtonGradientWrapper isRecording={props.isRecording}>
+      <RecordButtonInput onClick={props.handleRecordButtonClick}>
+        <RecordButtonIcon isRecording={props.isRecording} />
       </RecordButtonInput>
     </RecordButtonGradientWrapper>
   );
 }
 
+RecordButton.propTypes = {
+  isRecording: PropTypes.bool,
+  handleRecordButtonClick: PropTypes.func,
```

Comment:
```suggestion
  onRecordButtonClick: PropTypes.func,
```

The property or event is always named `on`, the actual function is named `handle`. 
This happens in multiple components.